misc/parent-pom/pom.xml

```diff
@@ -515,7 +515,7 @@
                 <plugin>
                     <groupId>org.jacoco</groupId>
                     <artifactId>jacoco-maven-plugin</artifactId>
-                    <version>0.8.7</version>
+                    <version>0.8.6</version>
```

Comment:
@metas-ts  downgrading jacoco plugin is intentional?

Reply:
no. please eave it at 0.8.7

Reply:
@riadmen  here we still have `0.8.6` instead of `0.8.7`.

@metas-ts  said
> please eave it at 0.8.7

so I understand we need `0.8.7`.